plutus-tx-plugin/test/Plugin/Primitives/deconstructorData2.plc.golden

```diff
@@ -289,7 +264,7 @@
             (termbind
               (strict)
               (vardecl
-                unsafeDataAsList_go
+                sfFromBuiltinBuiltinListNil_cfromBuiltin_go
```

Comment:
Similar to `patternMatchDo.plc.golden`. There are also some re-ordering of things in the PIR, but they look equivalent.